### PR TITLE
dev-inf: Add T-multitenant label to multi-tenant

### PR DESCRIPTION
**Pull request by ajstorm**
Added the t-multitenant lable to the multi-tenant section of TEAMS.yaml

Release note: None

Epic: None

### CODE REVIEW

**Review by knz — APPROVED** [review]
nice

you might want to remove triage_column_id since that refers to the now-obsolete board.

**Comment by ajstorm** [comment]
I removed the stale column tag.  TFTR!

bors r=knz

File: TEAMS.yaml [hunk 1]
```diff
@@ -79,7 +79,7 @@ cockroachdb/cluster-ui:
 cockroachdb/obs-inf-prs:
   triage_column_id: 14196277
 cockroachdb/multi-tenant:
-  triage_column_id: 16575985
+  label: T-multitenant
```
Comment (rafiss) [inline TEAMS.yaml:82]: does this have the intended effect? i would expect that you'd need to also update the `Team` struct in pkg/internal/team/team.go

Comment (rafiss) [reply TEAMS.yaml:82]: oh i see it now. i needed a rebase

**Pull request merged**